Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6064,16 +6064,18 @@ StorageAPIEnabled:
 
 StorageAPIEstimateEnabled:
   type: bool
-  status: testable
+  status: preview
```

Comment:
Do we want to limit the enablement to Safari Technology Preview and not stable/beta? Should be `stable` if this is ready for stable/beta.

Reply:
Yes, confirmed with Brent that setting status 'preview' with default value 'true' does not make much sense. I will change to stable.